src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -153,6 +157,16 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
+  <Target Name="_linkVSEmbeddableAssembly" AfterTargets="ResolveReferences" BeforeTargets="FindReferenceAssembliesForReferences">
+    <!-- This target can be deprecated when this issue is fixed:
+       https://github.com/NuGet/Home/issues/2365
+       and when the Microsoft.Internal.VisualStudio.Shell.Embeddable package have been updated to leverage the new feature.  -->
+    <ItemGroup>
+      <ReferencePath Condition=" '%(FileName)' == 'Microsoft.Internal.VisualStudio.Shell.Embeddable' ">
```

Comment:
Why is this not added in the embedinterop targets?

Reply:
I forgot you told me about that file. Interestingly, it's also already included.